lib/api/webSockets.js

```diff
@@ -76,7 +77,7 @@ const webSocketsFactory = (self) => {
 				const requestBody = logItem.request && logItem.request.requestBody;
 				const responseBody = logItem.response && logItem.response.responseBody;
 
-				if (config.logLevel === logLevels.debug) {
+				if (config.networkLogs === logLevels.debug || config.logLevel === logLevels.debug) {
```

Comment:
I think that specific log level should override general setting (have more weight). For example if `logLevel` defined `debug`, but `networkLogs` is not `debugg` (hidden for example) - network logs should not be displayed
@andriichumak what is preferable behavior?

Reply:
A agree